docs/konnectors.md

```diff
@@ -320,6 +320,56 @@ On the host create the file /etc/rkt/net.d/10-containers.conf
 
 and run your container with the "--net=bridge" option. That way, a new interface is available in the container and gives you access to the host.
 
+## Multi-account handling
+
+This section is devoted to allow the user to use one account for multiple konnectors. It will
+follow the following constraints in mind:
+
+- The migration path must be as easy as possible
+- The developpement and maintainance of konnector must also be as easy as possible
+
+### New doctype : io.cozy.konnector_account or io.cozy.account
+
+A new doctype will have to be create to allow to keep konnector accounts independently from each
+konnector.
+
+Maybe the existing io.cozy.account doctype, used by the email application, could also be used. Or
```

Comment:
We have no plan to port the emails app to cozy-v3 in the near future (2017). But it would be nice to use the same doctype for their accounts